docs/userguide/eng-image/dockerfile_best-practices.md

```diff
@@ -134,6 +134,41 @@ image. We recommend the [Debian image](https://hub.docker.com/_/debian/)
 since it’s very tightly controlled and kept minimal (currently under 150 mb),
 while still being a full distribution.
 
+### LABEL
+
+(Understanding Docker Object Labels)(../labels-custom-metadata.md)
+
+You can add labels to your image to help organize images by project, record
+licensing information, to aid in automation, or for other reasons. For each
+label, add a line beginning with `LABEL` and with one or more key-value pairs.
+The following examples show the different acceptable formats.
+
+>**Note**: If your string contains spaces, it must be quoted **or** the spaces
+must be escaped. If your string contains inner quote characters (`"`), escape
+them as well.
+
+```nohighlight
```

Comment:
Does this work currently (`nohighlight`?). I know we talked about implementing, so wondering if that was done

In this case, given that this is a Dockerfile example, perhaps better to change to "Dockerfile" to get the correct highlighting